service/history/stateBuilder.go

```diff
@@ -433,6 +433,9 @@ func (b *stateBuilderImpl) applyEvents(domainID, requestID string, execution sha
 			}
 			b.newRunTimerTasks = append(b.newRunTimerTasks, b.scheduleWorkflowTimerTask(event, newRunStateBuilder))
 
+			if newRunEventStoreVersion == persistence.EventStoreVersionV2 {
+				newRunStateBuilder.SetHistoryTree(*newExecution.RunId)
```

Comment:
Can you use getter instead of pointer dereference.  newExecution.GetRunId()

Reply:
ok